### PR TITLE
Adding PULL_REQUEST_TEMPLETE

### PR DESCRIPTION
**Pull request by kbagchiGWC**
Added `PULL_REQUEST_TEMPLETE`.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: PULL_REQUEST_TEMPLETE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- [Bulleted summary of changes]
+- [eg. Updated doc]
+- [eg. Added precondition to class]
+
+## Validation
+
+- [Bulleted summary of validation steps including local and CI pipeline]
+- [eg. Added new test]
+- [eg. Manually tested with app]
```
Comment (charliesantos) [inline PULL_REQUEST_TEMPLETE.md:24]:
```suggestion
- [eg. Manually tested with both app and server]
```

File: PULL_REQUEST_TEMPLETE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## Submission Checklist
+
+ - [ ] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code
+ - [ ] New unit tests and integration tests have beed added
+ - [ ] Code coverage is more or equal to the previous coverage
+ - [ ] A visual inspection of the `Files changed` tab was made prior to submitting the pull request ensuring the style guide was followed
+ - [ ] [CI pipeline](https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app) is green 
+
```
Comment (charliesantos) [inline PULL_REQUEST_TEMPLETE.md:8]:
```suggestion
- [ ] Included screenshot if necessary
```

**Review by charliesantos — COMMENTED** [review]
Left a couple of suggestions.

**Review by kpchoy — APPROVED** [review]

**Review by mhuynh5757 — APPROVED** [review]

**Review by charliesantos — APPROVED** [review]

**Comment by kbagchiGWC** [comment]
Green build 💯 .. merging now.

**Pull request merged**